docs/release-notes/release-notes-0.17.0.md

```diff
@@ -49,6 +49,16 @@ package](https://github.com/lightningnetwork/lnd/pull/7356)
   a helpful note-to-self containing arbitrary useful information about the
   channel.
 
+* [gRPC keepalive parameters can now be set in the
```

Comment:
Two suggestions for the release notes:
- Include all the params which have been added in the config (currently it only refers to the client setting for the keepalive)
- Mention that the grpc.keepalive_time_ms setting can be different depending on the programming language used for client development and developers should lookup the specific setting to utilize this feature

Reply:
It is my understanding that languages that link to the C based version of grpc are all going to be the same. The golang implementation of grpc seems to be the only other variant. Can anyone else confirm this?

Reply:
Added both suggestions, how does the new text look to you, @saubyk?

> The golang implementation of grpc seems to be the only other variant. Can anyone else confirm this?

Yes, the golang library unfortunately behaves differently from most others. I've only ever used JS or Python so can't say anything about the other ones.